networkgen/models.py

```diff
@@ -0,0 +1,41 @@
+from os import name
+import uuid
+from django.db import models
+from django.urls import reverse
+
+
+class Generator(models.Model):
+    """A model that holds parameters for the FEP network generator."""
+
+    SMILES = "SMILES"
+    MFP = "MFP"
+    Tanimoto = "Tanimoto"
+    MCS = "MCS"
+    METRICS_CHOICES = (
+        (SMILES, "SMILES"),
+        (MFP, "Morgan fingerprints (MFP)"),
+        (Tanimoto, "Tanimoto fingerprints (TFP)"),
+        (MCS, "Maximum common substructure (MCS)"))
+
+    uuid = models.UUIDField(
+        primary_key=True, default=uuid.uuid4, editable=False)
+    metric = models.CharField(
+        max_length=155, choices=METRICS_CHOICES, default=MFP)
+    created = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
```

Comment:
"created" and "modified" are going to be refactored from a external library (django-extensions TimeStampedModel).